template/en/default/admin/params/admin.html.tmpl

```diff
@@ -23,6 +23,14 @@
    desc = "Set up account policies"
 %]
 
+[% rate_limit_rules_desc = BLOCK %]
+This parameter is a json object that allows for overriding the default rate limit.
+Each key takes an array in the form of [MAX_RATE, SECONDS] where MAX_RATE is the maximum
```

Comment:
Consider some rewording here for flow:

"This parameter is a json object. It must have one or more valid keys, whose values are each an array [M,S]. There are two valid keys: <code>get; <code>show."